### ISSUE TITLE
ArgumentException when using split panes

### ISSUE THREAD

**Issue by angelobreuer** [natehitze/SeeUsingsLater#5]
Hello, since I am using the extension, I sporadically get the following error multiple times spammed:

```CSharp
System.ArgumentException: The specified ITextSnapshot doesn't belong to the correct TextBuffer
at Microsoft.VisualStudio.Text.Implementation.TrackingSpan.GetSpan(ITextSnapshot snapshot)   
at Microsoft.VisualStudio.Text.Implementation.TrackingSpan.GetStartPoint(ITextSnapshot snapshot)   
at SeeUsingsLater.SeeUsingsLaterTextViewCreationListener.Match(ICollapsible collapsible) in C:\Users\nate\Source\Repos\natehitze\SeeUsingsLater\SeeUsingsLater\SeeUsingsLaterTextViewCreationListener.cs:line 55   
at Microsoft.VisualStudio.Text.Outlining.OutliningManager.InternalCollapseAll(SnapshotSpan span, Predicate`1 match, Nullable`1 cancel)   
at Microsoft.VisualStudio.Text.Outlining.OutliningManager.CollapseAll(SnapshotSpan span, Predicate`1 match)  
at SeeUsingsLater.SeeUsingsLaterTextViewCreationListener.OnRegionsChanged(Object sender, RegionsChangedEventArgs regionsChangedEventArgs) in C:\Users\nate\Source\Repos\natehitze\SeeUsingsLater\SeeUsingsLater\SeeUsingsLaterTextViewCreationListener.cs:line 48   
at System.EventHandler`1.Invoke(Object sender, TEventArgs e)   
at Microsoft.VisualStudio.Text.Outlining.OutliningManager.UpdateAfterChange(NormalizedSnapshotSpanCollection changedSpans)   
at Microsoft.VisualStudio.Text.Outlining.OutliningManager.SourceTextChanged(Object sender, TextContentChangedEventArgs e)  
at Microsoft.VisualStudio.Text.Utilities.GuardedOperations.RaiseEvent[TArgs](Object sender, EventHandler`1 eventHandlers, TArgs args) 
--- End of stack trace from previous location where exception was thrown ---   
at Microsoft.VisualStudio.Telemetry.WindowsErrorReporting.WatsonReport.GetClrWatsonExceptionInfo(Exception exceptionObject)
```

__Steps to reproduce__
• Open Visual Studio
• Open a solution with one or multiple projects
• Open a file to the left and a file to the right (split-screen)
• Close Visual Studio
• Re-open Visual Studio
• Observe the error

Note that the error occurs mostly when using multiple code panes without having to restart Visual Studio.

___

This bug does not affect productivity for me (except that warning burning in the eyes), but I'm leaving this issue for information here.

I really appreciate any help you can provide.

___

Visual Studio Information:
Microsoft Visual Studio Community 2019 Preview
Version 16.9.0 Preview 1.0
VisualStudio.16.Preview/16.9.0-pre.1.0+30709.64
Microsoft .NET Framework
Version 4.8.04084
Installed Version: Community

**Comment by natehitze** [comment]
Thank you for the report. I've published a new version of the extension (1.7) to the marketplace. Please update and let me know how that works for you.

**Comment by angelobreuer** [comment]
Hello @natehitze,
Thank you a lot for your quick response.

The extension is now working fine in 1.7, and the `ArgumentException` no longer appears.
